src/xrCore/_std_extensions.cpp

```diff
@@ -13,29 +13,11 @@ int xr_strcmp(const char* S1, const char* S2)
 
 char* timestamp(string64& dest)
 {
-    string64 temp;
+    time_t     now = time(nullptr);
+    struct tm  tstruct;
+    tstruct = *localtime(&now);
 
-    /* Set time zone from TZ environment variable. If TZ is not set,
-    * the operating system is queried to obtain the default value
-    * for the variable.
-    */
-#if defined(WINDOWS)
-    _tzset();
-    u32 it;
+    strftime(dest, sizeof(dest), "%Y-%m-%d_%H-%M-%S", &tstruct);
```

Comment:
It better to use day-month-year format
```suggestion
    strftime(dest, sizeof(dest), "%d-%m-%y_%H-%M-%S", &tstruct);
```